mllib/src/main/scala/org/apache/spark/mllib/feature/Word2Vec.scala

```diff
@@ -158,6 +158,9 @@ class Word2Vec extends Serializable with Logging {
       .sortWith((a, b) => a.cn > b.cn)
     
     vocabSize = vocab.length
+    require(vocabSize > 0, "The vocabulary size should be large than 0. You may need to check " +
```

Comment:
Hm, I'd use `require` here. `IllegalArgumentException` is a `RuntimeException`, and I've always understood it to be slightly bad style to throw `RuntimeException` as you can't catch for it with catching for any `RuntimeException`.